formatter.js

```diff
@@ -0,0 +1,51 @@
+/*
+  Custom eslint formatter for eslint-plugin-i18n-json to allow better error message display.
+  Heavily inspired from https://github.com/sindresorhus/eslint-formatter-pretty.
+*/
+
+const chalk = require('chalk');
+const plur = require('plur');
+const logSymbols = require('log-symbols');
+const indentString = require('indent-string');
+const path = require('path');
+
+const CWD = process.cwd();
+
+const formatter = (results) => {
+  const total = {
+    errors: 0,
+    warnings: 0
+  };
+  const outputs = results.map(({ filePath, messages , errorCount, warningCount}) => {
```

Comment:
Did you run linters over this file?  There's a few style issues I'd expect a linter to pick up.

Run the linters :)

Reply:
thanks, must have left this out from being linted